module/VuFind/src/VuFind/ILS/Driver/Aleph.php

```diff
@@ -1284,52 +1284,62 @@ public function getMyHolds($user)
             $delete = $item->xpath('@delete');
             $href = $item->xpath('@href');
             $item_id = substr($href[0], strrpos($href[0], '/') + 1);
-            if ((string)$z37->{'z37-request-type'} == "Hold Request" || true) {
-                $type = "hold";
-                //$docno = (string) $z37->{'z37-doc-number'};
-                //$itemseq = (string) $z37->{'z37-item-sequence'};
-                $seq = (string)$z37->{'z37-sequence'};
-                $location = (string)$z37->{'z37-pickup-location'};
-                $reqnum = (string)$z37->{'z37-doc-number'}
-                    . (string)$z37->{'z37-item-sequence'}
-                    . (string)$z37->{'z37-sequence'};
-                $expire = (string)$z37->{'z37-end-request-date'};
-                $create = (string)$z37->{'z37-open-date'};
-                $holddate = (string)$z37->{'z37-hold-date'};
-                $title = (string)$z13->{'z13-title'};
-                $author = (string)$z13->{'z13-author'};
-                $isbn = (string)$z13->{'z13-isbn-issn'};
-                $barcode = (string)$z30->{'z30-barcode'};
-                $status = (string)$z37->{'z37-status'};
-                if ($holddate == "00000000") {
-                    $holddate = null;
-                } else {
-                    $holddate = $this->parseDate($holddate);
-                }
-                $delete = ($delete[0] == "Y");
-                // Secondary, Aleph-specific identifier that may be useful for
-                // local customizations
-                $adm_id = (string)$z30->{'z30-doc-number'};
-
-                $holdList[] = [
-                    'type' => $type,
-                    'item_id' => $item_id,
-                    'adm_id'   => $adm_id,
-                    'location' => $location,
-                    'title' => $title,
-                    'author' => $author,
-                    'isbn' => $isbn,
-                    'reqnum' => $reqnum,
-                    'barcode' => $barcode,
-                    'id' => $this->barcodeToID($barcode),
-                    'expire' => $this->parseDate($expire),
-                    'holddate' => $holddate,
-                    'delete' => $delete,
-                    'create' => $this->parseDate($create),
-                    'status' => $status,
-                    'position' => ltrim($seq, '0')
-                ];
+            $type = "hold";
+            $seq = null;
+            $item_status = preg_replace(
+                "/\s[\s]+/",
+                " ",
+                (string)$item->{'status'}
+            );
+            if (preg_match(
+                "/Waiting in position ([0-9]+) in queue; "
```

Comment:
Is this guaranteed to be consistent, or might it vary based on i18n or Aleph settings?

Reply:
It may depend on language in Aleph and even Aleph administrators can change the text in translation files. Maybe we should make it configurable?

Reply:
I made it configurable.